java/ql/src/Compatibility/JDK9/JdkInternalsReplacement.qll

```diff
@@ -11,18 +11,21 @@ predicate jdkInternalReplacement(string old, string new) {
 }
 
 private predicate jdkInternalReplacement(string r) {
-  r = "com.sun.crypto.provider.SunJCE=Use java.security.Security.getProvider(provider-name) @since 1.3" or
+  r =
+    "com.sun.crypto.provider.SunJCE=Use java.security.Security.getProvider(provider-name) @since 1.3" or
   r = "com.sun.org.apache.xml.internal.security=Use java.xml.crypto @since 1.6" or
   r = "com.sun.org.apache.xml.internal.security.utils.Base64=Use java.util.Base64 @since 1.8" or
   r = "com.sun.org.apache.xml.internal.resolver=Use javax.xml.catalog @since 9" or
   r = "com.sun.net.ssl=Use javax.net.ssl @since 1.4" or
-  r = "com.sun.net.ssl.internal.ssl.Provider=Use java.security.Security.getProvider(provider-name) @since 1.3" or
+  r =
+    "com.sun.net.ssl.internal.ssl.Provider=Use java.security.Security.getProvider(provider-name) @since 1.3" or
```

Comment:
These lines are tricky since the string literal alone takes them over the maximum line length that the autoformatter tries to achieve, so any opportunity given to the autoformatter to make the lines shorter will be taken.